core/src/main/scala/com/stripe/agate/tensor/Storage.scala

```diff
@@ -102,14 +102,14 @@ object Storage {
       }
 
     def copyToArrayStorage: Storage.ArrayStorage[Float] = {
-      val len = (buffer.limit - offset).toLong
+      val len = (buffer.limit() - offset).toLong
       val writable = StorageAllocator[Float].allocate(len)
       writeInto(writable, 0L, Shape.SingleDim, len)
       writable.toArrayStorage
     }
 
     def size: Long =
-      buffer.limit - offset.toLong
+      buffer.limit() - offset.toLong
```

Comment:
`test` doesn't work for me. The default scala version complains that it can't tell which `limit` is being called. One with and one without an arg. This fixed it.